doc/install/install-kubernetes.md

```diff
@@ -0,0 +1,120 @@
+# Install Kuadrant on a Kubernetes cluster
+
+> [!NOTE]  
+> You must perform these steps on each Kubernetes cluster where you want to use Kuadrant.
+
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, with `kubeadmin` or an account with similar permissions
+
+## Procedure
+
+This guide will show you how to install Kuadrant onto a bare Kubernetes cluster.
+
+Alternatively, if you are looking instead for a way to set up Kuadrant locally to evaluate or develop, consider running the kind & Kubernetes [quickstart script](https://docs.kuadrant.io/latest/getting-started-single-cluster/).
+
+### Install Gateway API
+
+```bash
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+```
+
+### Install [OLM](https://olm.operatorframework.io/)
+
+```bash
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.23.1/install.sh | bash -s v0.23.1
+```
+
+### Install Istio as a Gateway API provider
+
+> [!NOTE]  
+> There are several ways to install Istio (via `istioctl`, Helm chart or Operator) - this is just an example for starting from a bare Kubernetes cluster.
+
+
+```bash
+curl -sSL https://istio.io/downloadIstio | ISTIO_VERSION=1.20.7 sh -
```

Comment:
Should we specify a later version since 1.20 is out of support since Jun 25, 2024?

```shell
$ ./istio-1.20.7/bin/istioctl install --set profile=minimal -y
WARNING: Istio 1.20.0 may be out of support (EOL) already: see https://istio.io/latest/docs/releases/supported-releases/ for supported releases
```

Reply:
bumped to `1.21.4`